assets/js/modules/adsense/index.legacy.js

```diff
@@ -1,63 +0,0 @@
-/**
```

Comment:
Looks like we can also delete the `parseAFCClientID` function in the `assets/js/modules/adsense/util/parsing.js` file because we don't use it anymore.